Xamarin.Forms.Material.Android/MaterialColors.cs

```diff
@@ -8,16 +8,23 @@
 using ASeekBar = Android.Widget.AbsSeekBar;
 using PlatformColor = Android.Graphics.Color;
 using Xamarin.Forms.Platform.Android;
-#else
+#elif __IOS__
```

Comment:
are we sure about this ? iOS does uses this project ?

Reply:
well... I think so. I just followed the existing condition. 😉